glean-core/ios/Glean/Glean.swift

```diff
@@ -28,6 +28,7 @@ public class Glean {
     var initialized: Bool = false
     private var uploadEnabled: Bool = true
     private var debugViewTag: String?
+    var logPings: Bool?
```

Comment:
This probably could have been left in the configuration and made internal, but I'm okay with this being a member variable of Glean.

Reply:
I was just following the same method we used for the `uploadEnabled` and the `debugViewTag`. The only reason it is not also private is so that we can look at this property in the tests.

Reply:
If you have any ideas on how to test without making this public, that might be best.

Reply:
The default of `internal` here is fine, it allows us to access it in tests and it doesn't expose it to the public API.